feeder/parsers/PL.py

```diff
@@ -0,0 +1,69 @@
+
+from bs4 import BeautifulSoup
+import requests
+import arrow
+
+COUNTRY_CODE = 'PL'
+
+def fetchValue(params):
+
+    now = arrow.utcnow()
+    end = now.replace(hours=+2)
+    start = now.replace(hours=-22)
+    periodEnd = str(end.format('YYYYMMDDHH00'))
+    periodStart = str(start.format('YYYYMMDDHH00'))
+
+
+
+    parameters = '&psrType=' + params + '&documentType=A75&processType=A16&in_Domain=10YPL-AREA-----S&periodStart=' + periodStart + '&periodEnd=' + periodEnd
+    link = 'https://transparency.entsoe.eu/api?securityToken=7466690c-c66a-4a00-8e21-2cb7d538f380' + parameters
+
+    content = requests.get(link)
```

Comment:
This will create a new HTTP connection everytime.
What you can do is, outside of this function, create a session using `session = requests.session()`. Then inside the function you can re-use the same session using `session.get(link)`